cmd/resumic/validate/doc.go

```diff
@@ -0,0 +1 @@
+package validate
```

Comment:
Why the validate/doc.go file?

Reply:
@stp thought to use it for comments that might be useful for the godoc later.